qwik-graphql-tailwind/src/components/issue-pr-card/card.tsx

```diff
@@ -0,0 +1,72 @@
+import { component$ } from '@builder.io/qwik';
+import { format } from 'date-fns';
+import { IssuesIcon, PullRequestIcon } from '../icons';
+import * as styles from './issue-pr-card.classNames';
+
+export interface IssuePrCardProps {
+  data: {
+    url: string;
+    title: string;
+    number: number;
+    isOpen: boolean;
+    createdAt: string;
+    authorName: string;
+    commentsCount: number;
+  };
+  type: 'issue' | 'pr';
+}
+
+export const IssuePrCard = component$(({ data, type }: IssuePrCardProps) => {
+  return (
+    <div className={styles.card_container}>
+      <div className="flex">
+        <label className={styles.card_checkbox}>
+          <input type="checkbox" name="issues[]" autoComplete="off" />
+        </label>
+
+        <div className="flex-shrink-0 pl-4">
+          <span>{type == 'issue' ? <IssuesIcon className="w-5 h-5" /> : <PullRequestIcon className="w-5 h-5" />}</span>
```

Comment:
this icons varies, please check. there are different icons closed PR/ISSue and the colors for every state are different

Reply:
Yes when the issue or PR is closed the icon changes, will add that also.